src/inject.js

```diff
@@ -137,7 +146,7 @@ const stringHandlers = {
 let alreadyInjected = {};
 
 // This is the buffer of styles which have not yet been flushed.
-let injectionBuffer = "";
+let injectionBuffer = [];
```

Comment:
Could you give this a `: string[]` type to help flow make sure that these strings can't be null?